<launch>

  <!-- Launch block grasp generator -->
  <node name="grasp_generator_server" pkg="block_grasp_generator" type="block_grasp_generator_server"/>
  <!-- Launch grasping action server -->
  <node name="grasp_object_server" pkg="reem_tabletop_grasping" type="reem_object_manipulation_as.py"/>
  <!-- Launch the depth throtling service -->
  <node name="depth_throtle_server" pkg="reem_tabletop_grasping" type="depth_throtle_server.py"/>
  <!-- Launch the clusters detection -->
  <node name="clusters_detection" pkg="object_recognition_core" type="detection" args="-c $(find reem_object_recognition)/config/tabletop/detection.clusters.ros.ork.reem.throtled" />

</launch>
